mne/channels/channels.py

```diff
@@ -206,6 +208,83 @@ def equalize_channels(instances, copy=True, verbose=None):
     return equalized_instances
 
 
+def unify_bad_channels(insts):
+    """Unify bad channels across a list of instances.
+
+    All instances must be of the same type and have matching channel names and channel
+    order. The ``.info["bads"]`` of each instance will be set to the union of
+    ``.info["bads"]`` across all instances.
+
+    Parameters
+    ----------
+    insts : list
+        List of instances (:class:`~mne.io.Raw`, :class:`~mne.Epochs`,
+        :class:`~mne.Evoked`, :class:`~mne.time_frequency.Spectrum`,
+        :class:`~mne.time_frequency.EpochSpectrum`) across which to unify bad channels.
```

Comment:
CircleCI treats warnings as errors when building docs so if you search "warning" you'll see in [the log](https://app.circleci.com/pipelines/github/mne-tools/mne-python/20948/workflows/bdb1316d-4e09-42bc-85f7-961b6d566f7a/jobs/59502):

```
/home/circleci/project/mne/channels/channels.py:docstring of mne.channels.channels.unify_bad_channels:11: WARNING: py:class reference target not found: mne.time_frequency.EpochSpectrum
```
so
```suggestion
        :class:`~mne.time_frequency.EpochsSpectrum`) across which to unify bad channels.
```